src/wp-admin/includes/upgrade.php

```diff
@@ -272,14 +272,16 @@ function wp_install_defaults( $user_id ) {
 		$first_comment_email  = ! empty( $first_comment_email ) ? $first_comment_email : 'wapuu@wordpress.example';
 		$first_comment_url    = ! empty( $first_comment_url ) ? $first_comment_url : esc_url( __( 'https://wordpress.org/' ) );
 		$first_comment        = ! empty( $first_comment ) ? $first_comment : sprintf(
-			/* translators: %s: Gravatar URL. */
+			/* translators: %1$s: Gravatar URL, %2$s: Learn WordPress URL */
 			__(
 				'Hi, this is a comment.
-To get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.
-Commenter avatars come from <a href="%s">Gravatar</a>.'
+				 To get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.
+				 Commenter avatars come from <a href="%1$s">Gravatar</a>. Click here to <a href="%2$s">Learn WordPress</a>.'
```

Comment:
I think we should split this out onto a new line and add a little more context to the link.

```suggestion
				 Commenter avatars come from <a href="%1$s">Gravatar</a>. 
				 If you're new to WordPress, check at these free <a href="%2$s">online courses at Learn WordPress</a>.'
```